.github/workflows/release-quil-py.yml

```
@@ -0,0 +1,109 @@
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Release tag starts with quil-py/v, proceeding with release"
+
+  macos:
+    runs-on: macos-11
+    needs: is-python-release
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11']
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Build wheels - universal2
+        # universal2 supports both x86_64 and aarch64
+        uses: messense/maturin-action@v1
+        with:
+          args: -i python --release --universal2 --manifest-path quil-py/Cargo.toml --out dist
+      - name: Install wheel
+        run: |
+          pip install quil --no-index --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: dist
+
+  linux:
+    runs-on: ubuntu-latest
+    needs: is-python-release
+    env:
+      CXXFLAGS: "-std=c++11"
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11']
+        target: [x86_64, aarch64]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        default: true
+    - uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Build wheels
+      uses: messense/maturin-action@v1
+      with:
+        target: ${{ matrix.target }}
+        manylinux: 2014
+        args: -i ${{ matrix.python-version }} --release --manifest-path quil-py/Cargo.toml --out dist
+    - name: Upload wheels
+      uses: actions/upload-artifact@v3
+      with:
+        name: wheels
+        path: dist
+
+  publish-python-package:
+    name: Release
+    runs-on: ubuntu-latest
+    needs: [ macos, linux ]
+    steps:
+      - uses: actions/download-artifact@v3
+      - name: Publish to PyPi
+        env:
+          MATURIN_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          MATURIN_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        uses: messense/maturin-action@v1
+        with:
+          command: upload 
+          args: --skip-existing wheels/*
+
+  publish-rust-crate:
+    name: Release
+    runs-on: ubuntu-latest
+    needs: [ publish-python-package ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.PAT }}
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - run: cargo publish --manifest-path=quil-rs/Cargo.toml --token ${{ secrets.CRATES_IO_TOKEN }}
```

I think this is publishing the rust library again instead of the pyo3 one

```suggestion
      - run: cargo publish --manifest-path=quil-py/Cargo.toml --token ${{ secrets.CRATES_IO_TOKEN }}
```

---

Nice catch, fixed.